### ISSUE TITLE
Better handling for no found test functions in schema.

### ISSUE THREAD

**Issue by michaeljota** [scottwrobinson/camo#41]
Right now if you put some invalid validator in the schema, it throws a not found error.

```javascript
that._schema[key].match.test is not a function
```

In the example, 'match' it's the name of the invalid validator. 

I guess it should throw some information about where it is being used.

**Comment by scottwrobinson** [comment]
Yes, there is a lot of room for improvement on the errors it throws. Most aren't descriptive enough.

I've added this to my to-do list. Thanks for pointing it out!
